### PR TITLE
add mainframe

### PR DESCRIPTION
**Pull request by a29paul**

### CODE REVIEW

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import SetttingsIcon from '@material-ui/icons/Settings';
+import { withStyles } from "@material-ui/styles";
+import theme from '../../../theme';
+import styles from "../../auth/styles";
+
+class MainFrame extends Component<any,any> {
+    public render(){
+    const { classes } = this.props;
+    return (
+
```
Comment (virajg224) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: remove line

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                >
+                    <div className={classes.toolbar} />
+                    <List>
+                        <ListItem button key='Files'>
+                            <ListItemIcon> 
+                                <FileCopyIcon /> 
+                            </ListItemIcon>
+                            <ListItemText primary='Files' />
+                        </ListItem>
+                        <ListItem button key='Groups'>
```
Comment (virajg224) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: Please refer to mockup for required tabs

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                </Paper>
+
+            </div>
+        </body>
+
+
+    );
+   }
+}
+export default withStyles(styles)(MainFrame)
```
Comment (virajg224) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: [Minor] EOF line

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import GroupIcon from '@material-ui/icons/Group';
+import SetttingsIcon from '@material-ui/icons/Settings';
+import { withStyles } from "@material-ui/styles";
+import theme from '../../../theme';
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: remove unused imports

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                        <Typography variant="h6" noWrap>
+                            Contrail Web
+            </Typography>
+                    </Toolbar>
+                </AppBar>
+                <Drawer
+                    className={classes.drawer}
+                    variant="permanent"
+                    classes={{
+                        paper: classes.drawerPaper,
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: make it one line please

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                            <ListItemIcon> 
+                                <FileCopyIcon /> 
+                            </ListItemIcon>
+                            <ListItemText primary='Files' />
+                        </ListItem>
+                        <ListItem button key='Groups'>
+                            <ListItemIcon> 
+                                <GroupIcon /> 
+                            </ListItemIcon>
+                            <ListItemText primary='Groups' />
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: Change all to quotes ", not '

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+                            <div>
+                                <h1> Welcome to the MainFrame!</h1>
+                            </div>
+                        </Box>
+                    </Container>
+                </Paper>
+
+            </div>
+        </body>
+
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: remove extra lines

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+import Paper from '@material-ui/core/Paper';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import GroupIcon from '@material-ui/icons/Group';
+import SetttingsIcon from '@material-ui/icons/Settings';
+import { withStyles } from "@material-ui/styles";
+import theme from '../../../theme';
+import styles from "../../auth/styles";
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: Why are you using my styles lol

You should copy the styles from your template and put it in a new file, and then import that one instead

File: contrail-ui/src/components/main/mainframe/MainFrame.tsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import Container from '@material-ui/core/Container';
+import AppBar from '@material-ui/core/AppBar';
+import Drawer from '@material-ui/core/Drawer';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import Box from '@material-ui/core/Box';
```
Comment (1298se) [inline contrail-ui/src/components/main/mainframe/MainFrame.tsx]: alphabetize imports.

**Comment by 1298se** [comment]
Closed because duplicate in #54

**Pull request closed without merging**